package.json

```diff
@@ -269,13 +264,6 @@
         "category": "Daffodil Debug",
         "enablement": "inDebugMode"
       },
-      {
-        "command": "hexview.display",
-        "title": "Display the hex view",
-        "category": "Daffodil Debug",
-        "enablement": "inDebugMode",
-        "icon": "$(file-binary)"
-      },
```

Comment:
I think this should stay in but update it to use `data.edit` instead of `hexview.display` and update the title. It would still be nice to click this icon and have it just open the data editor.

Reply:
Okay, I'll have to take a look into this and see if anything would need to be refactored since the data editor requires either a filename / file selection on initialization.

Reply:
I think it would make sense if its going to involve any significant refactoring to happen in its own issue and have this PR get merged without that functionality